package.json

```diff
@@ -1,17 +1,33 @@
 {
-  "name": "flat-earth",
+  "name": "@map-colonies/flat-earth",
```

Comment:
Now is the time to talk about the name. What about something like `geo-tools`?

Reply:
`geo-kitty` or `geokitty`